src/index.css

```diff
@@ -42,4 +42,21 @@ div.App form {
 
 .required-text {
 	color: red;
-}
\ No newline at end of file
+}
+
+
+.form-check {
```

Comment:
I still see regular-sized check boxes on my laptop. Let's create new classes instead of overwriting the default class.
<img width="639" alt="Screen Shot 2019-05-27 at 5 03 43 PM" src="https://user-images.githubusercontent.com/4501235/58442667-802c7d00-80a1-11e9-8f6e-a3f3fb696b06.png">